implementations/rust/ockam/ockam_command/src/enroll/mod.rs

```diff
@@ -38,6 +44,8 @@ impl EnrollCommand {
             AuthenticateEnrollmentTokenCommand::run(opts, cmd)
         } else if cmd.auth0 {
             EnrollAuth0Command::run(opts, cmd)
+        } else if cmd.email.is_some() {
+            DevEnrollCommand::run(opts, cmd)
```

Comment:
@polvorin do we need this command in the production binary? could we put this entire functionality behind a feature flag?

Reply:
yes good point,  and will remove it entirely before merging actually,  I can use a different side channel to prepopulate authenticated users for integration test